app/views/spree/orders/_bought.html.haml

```diff
@@ -16,10 +16,10 @@
       %td.cart-item-description
 
         %div.item-thumb-image
-          - if variant.images.length == 0
-            = mini_image(variant.product)
+          - if variant.product.images.length == 0
+            = image_tag("/noimage/mini.png")
```

Comment:
AFAIK the current status re variant images is that variant specific images can be added to the system in the product images page BUT they are not used anywhere. SO, we can make all our code go to variant.product.images ONLY. Which is what you are doing here. So I think this is ok, just sharing my thoughts :-)